utils/pod/label.go

```diff
@@ -0,0 +1,13 @@
+package pod
+
+import (
+	"github.com/paypal/dce-go/types"
+)
+
+func GetServiceDetail() types.ServiceDetail {
```

Comment:
Just a suggestion on the naming, take it if make sense to you. label.go could be confused as in docker-compose there is a specific section called label. But here `ServiceDetail ` is updating all field, not only label. Maybe `service.go` explains better on what this file is doing. And variable `var ServiceDetail = make(map[string]map[string]interface{})` could move here as well.

Reply:
done